crates/epaint/src/tessellator.rs

```diff
@@ -470,7 +470,7 @@ impl Path {
                 self.add_point(points[i], n1c / n1c.length_sq());
             } else {
                 // miter join
-                self.add_point(points[i], normal / length_sq);
+                self.add_point(points[i], normal.normalized());
```

Comment:
I believe the fix is wrong. Dividing with length-squared is what gets the correct widths for a miter-join. If that sometimes causes numerical problems we should either fix those narrow cases (if angle < …) or switch from miter-joins

Reply:
Oh sorry, I didn't realize that, I just changed it thinking normal vectors *always* have unit length, and this fixed my problem with rendering.

The problem appeared when I tried to render points distributed this way (this is the bottom part of the image above): 
![points](https://github.com/user-attachments/assets/adc04855-cc94-4927-8435-21ea45ca7b0d)

Reply:
Oh I just read the issue linked in the code. I guess i should have done a little bit more research before opening the PR.